.github/workflows/python_tests.yml

```diff
@@ -0,0 +1,25 @@
+name: python_tests
+on: [push]  # pull_request,
+jobs:
+  python_tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7, 3.8]  # , 3.9]
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: sudo apt-get install libxml2-dev libxslt-dev
```

Comment:
Can you add a comment with why these installs are necessary?

Reply:
DOne.